Sources/SWBTaskConstruction/TaskProducers/OtherTaskProducers/CustomTaskProducer.swift

```diff
@@ -34,28 +34,34 @@ final class CustomTaskProducer: PhasedTaskProducer, TaskProducer {
                 let workingDirectory = customTask.workingDirectory.map { Path(context.settings.globalScope.evaluate($0)).normalize() } ?? context.defaultWorkingDirectory
                 let inputPaths = customTask.inputFilePaths.map { Path(context.settings.globalScope.evaluate($0)).normalize() }
                 let inputs = inputPaths.map { delegate.createNode($0) }
-                var outputs: [any PlannedNode] = customTask.outputFilePaths.map { Path(context.settings.globalScope.evaluate($0)).normalize() }.map { delegate.createNode($0) }
-                
+                let outputPaths = customTask.outputFilePaths.map { Path(context.settings.globalScope.evaluate($0)).normalize() }
+                var outputs: [any PlannedNode] = outputPaths.map { delegate.createNode($0) }
+
+                let md5Context = InsecureHashContext()
```

Comment:
It probably won't matter in practice most of the time, but the variable-length hash components really _should_ be framed (e.g. to ensure inputPaths=a, outputPaths=b,c,d is hashed differently than inputPaths=a,b outputPaths=c,d)